src/resolvers/dockerfileTemplate.ts

```diff
@@ -78,6 +79,7 @@ export class DockerfileTemplateResolver implements Resolver {
 			RESIN_MACHINE_NAME: bundle.deviceType,
 			BALENA_ARCH: bundle.architecture,
 			BALENA_MACHINE_NAME: bundle.deviceType,
+			...additionalTemplateVars,
```

Comment:
Putting this at the end allows overriding the default ones of BALENA_ARCH/BALENA_MACHINE_NAME/etc, I assume that's fine but worth being aware of

Reply:
Yeah shouldn't be a problem, but good point